### DISCUSSION TITLE
Where are the up-to-date docs for OrientDB?

### DISCUSSION THREAD

**Discussion by smolinari** [orientechnologies/orientdb#9720] [Q&A] [answered]
Hi,

Just wondering. Where are the up-to-date docs?

Scott

**Comment by luigidellaquila** [comment, marked as answer]
Hi Scott,

I'd suggest to check this

https://orientdb.org/docs/3.2.x/

Thanks

Luigi

**Reply by smolinari** [reply]
@luigidellaquila - Thanks Luigi! Hey. How are things going with the OrientDB team? It's been quite a while since I've been active in the community and I've sort of only been watching things out from the fringe. I popped in and wanted to check out the advancements and couldn't find the up-to-date docs. The link in Github only goes to a "We're now part of SAP" page (congrats on that btw), and Google only got me to 3.1.0-beta docs. 😀 

Hope you all are doing well. 

Scott
